Mathlib/NumberTheory/LSeries/HurwitzZetaEven.lean

```diff
@@ -212,7 +212,10 @@ lemma hasSum_nat_cosKernel₀ (a : ℝ) {t : ℝ} (ht : 0 < t) :
     mul_zero, zero_mul, Complex.exp_zero, Real.exp_zero, ofReal_one, mul_one, Int.cast_neg,
     Int.cast_natCast, neg_sq, ← add_mul, add_sub_assoc, ← sub_sub, sub_self, zero_sub,
     ← sub_eq_add_neg, mul_neg] at this
-  convert this with n
+  -- workaround for a slow `convert`
+  -- should this be extracted as a congruence lemma for `HasSum`?
+  have hh {f g : ℕ → ℂ} {z : ℂ} (hf : HasSum f z) (h : g = f) : HasSum g z := h ▸ hf
+  refine hh this <| funext fun n ↦ ?_
```

Comment:
Now you can use the new congruence lemma here.

Reply:
Of course. I was just waiting for CI to build all the files in between (I should probably have applied the awaiting-CI tag.) Done now.